packages/grafana-ui/src/components/Link/TextLink.story.tsx

```diff
@@ -0,0 +1,71 @@
+import { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+
+import { StoryExample } from '../../utils/storybook/StoryExample';
+import { VerticalGroup } from '../Layout/Layout';
+
+import { TextLink } from './TextLink';
+import mdx from './TextLink.mdx';
+
+const meta: Meta = {
+  title: 'General/TextLink',
+  component: TextLink,
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+    controls: { exclude: ['href', 'external'] },
+  },
+  argTypes: {
+    variant: { control: 'select', options: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'body', 'bodySmall', undefined] },
+    weight: {
+      control: 'select',
+      options: ['bold', 'medium', 'light', 'regular', undefined],
+    },
+    color: {
+      control: 'select',
+      options: ['primary', 'secondary', 'disabled', 'link', 'maxContrast', undefined],
+    },
+    inline: { control: 'boolean' },
+  },
+};
+
+export const Example: StoryFn = () => {
```

Comment:
I had it like this:
```
Example.parameters = {
 controls: { exclude: ['href', 'external', 'variant', 'weight', 'color', 'inline', 'icon'] },
};
```
But I realised it didn't change anything, it keeps showing this message. Just in case, I will check if I'm missing something.
<img width="523" alt="Captura de pantalla 2023-06-29 a las 19 17 24" src="https://github.com/grafana/grafana/assets/65417731/4c34e146-23f9-4ef3-9f47-865abb8337cb">